packages/aws-client/src/S3.ts

```diff
@@ -199,13 +202,6 @@ export const s3ObjectExists = (params: { Bucket: string, Key: string }) =>
 
 /**
  * Wait for an object to exist in S3
- *
- * @param {Object} params
- * @param {string} params.bucket
- * @param {string} params.key
- * @param {number} [params.interval=1000] - interval before retries, in ms
- * @param {number} [params.timeout=30000] - timeout, in ms
- * @returns {Promise<undefined>}
  */
 export const waitForObjectToExist = async (params: {
```

Comment:
Nit: we should add in what the interval/timeout numbers mean (e.g. it should probably still specify ms)